spec/paypalhttp/encoder_spec.rb

```diff
@@ -116,13 +116,46 @@
           "content-type" => "application/x-www-form-urlencoded; charset=utf8"
         },
         :body => {
-          :key => " ..<..>..%..{..}..|../..^..`..!",
+          :key => " ..<..>..%..{..}..|..^..`",
           :another_key => 1013,
         }
       })
       serialized = Encoder.new.serialize_request(req)
 
-      expect(serialized).to eq("key=+..%3C..%3E..%25..%7B..%7D..%7C..%2F..%5E..%60..%21&another_key=1013")
+      expect(serialized).to eq("key=%20..%3C..%3E..%25..%7B..%7D..%7C..%5E..%60&another_key=1013")
+    end
+
+    it 'encodes different special/unsafe characters when using CGI.escape' do
```

Comment:
This seems like a duplicate of the above test case. Make sure to also change the test name from mentioning CGI to mentioning URI Parser